c-dependencies/js-compute-runtime/js-compute-builtins.cpp

```diff
@@ -6690,7 +6742,11 @@ bool searchParams_get(JSContext *cx, unsigned argc, Value *vp) {
   RootedObject params(cx);
   if (params_val.isNullOrUndefined()) {
     JSUrl *url = (JSUrl *)JS::GetReservedSlot(self, Slots::Url).toPrivate();
-    params = URLSearchParams::create(cx, url);
+    RootedObject url_search_params_intance(
+        cx, JS_NewObjectWithGivenProto(cx, &URLSearchParams::class_, URLSearchParams::proto_obj));
+    if (!self)
+      return false;
```

Comment:
I'm not sure this is necessary here -- line 6744 is using `JS::GetReservedSlot` on `self`.

Reply:
good spot, this was meant to be `url_search_params_instance`